### PR TITLE
Feature/qa dashboard

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please see order by commits

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 1]
```diff
@@ -24,6 +24,7 @@
         >
             <p-text-button icon-left="ic_refresh"
                            style-type="highlight"
+                           :disabled="!isFormDataChanged"
                            class="return-to-initial-settings-button"
                            @click="handleReturnToInitialSettings"
             >
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:30]: description: disable button when options are not changed

스크린샷 2023-08-08 오후 1 39 58

File: apps/web/src/services/dashboards/widgets/_components/WidgetViewModeSidebar.vue [hunk 2]
```diff
@@ -44,7 +44,13 @@
...
                 <div class="non-inherited-option-modal-body">
                     <p>
-                        {{ $t('DASHBOARDS.FULL_SCREEN_VIEW.APPLY_NON_INHERITED_OPTION_HELP_TEXT') }}
+                        <p-i name="ic_warning-filled"
+                             color="inherit"
+                             width="1rem"
+                             height="1rem"
+                             class="warning-icon"
+                        />
+                        <span>{{ $t('DASHBOARDS.FULL_SCREEN_VIEW.APPLY_NON_INHERITED_OPTION_HELP_TEXT') }}</span>
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_components/WidgetViewModeSidebar.vue:53]: description:

스크린샷 2023-08-08 오후 1 40 55

**Review by skdud4659 — APPROVED** [review]
LGTM~!

File: apps/web/src/services/dashboards/shared/dashboard-variables/DashboardVariablesSelectDropdown.vue [hunk 3]
```diff
@@ -7,15 +7,19 @@
...
             </div>
         </template>
-        <dashboard-variables-more-button :is-manageable="props.isManageable" />
+        <dashboard-variables-more-button :is-manageable="props.isManageable"
+                                         :disabled="state.saveLoading"
+        />
         <p-text-button style-type="highlight"
                        class="reset-button"
+                       :disabled="state.saveLoading"
                        @click="dashboardDetailStore.resetVariables(props.originVariables, props.originVariablesSchema)"
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-variables/DashboardVariablesSelectDropdown.vue:23]: https://github.com/cloudforet-io/console/assets/18563857/38f13e33-db16-40f8-ad75-6724be61ac9e

Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-variables/DashboardVariablesSelectDropdown.vue:23]: https://github.com/cloudforet-io/console/assets/18563857/a6f22cec-b1e9-4f38-b80d-839d83cd7b13

**Pull request merged**